### ISSUE TITLE
.status.loadBalancer.ingress[0].hostname of networking.k8s.io/v1:Ingress

### ISSUE THREAD

**Issue by Iced-Sun** [pulumi/pulumi-kubernetes#1498]
The code
```
const ingress = new k8s.networking.v1.Ingress(...);
ingress.status.loadBalancer.ingress[0].hostname.apply(n => console.log(n));
```
produces output of `undefined`, while 
```
const ingress = new k8s.extensions.v1beta1.Ingress(...);
ingress.status.loadBalancer.ingress[0].hostname.apply(n => console.log(n));
```
is OK.

## Expected behavior

Both version of `Ingress` api should await the ingress to be ready (i.e., `.status.loadBalancer.ingress[0].hostname` is available) and output the hostname of created lb.

## Current behavior

The api `k8s.networking.v1.Ingress` doesn't await ingress ready. The related code should lie [here](https://github.com/pulumi/pulumi-kubernetes/blob/80d4c16c2240d77ca575273a9c167c739e7681a3/provider/pkg/await/awaiters.go#L117), in which `networking.k8s.io/v1:Ingress` is apparently not included.

**Comment by viveklak** [comment]
This seems to be one of the contributing issues to https://github.com/pulumi/pulumi-kubernetes/issues/1649

**Comment by viveklak** [comment]
Fixed in #1795. Will be in v3.10.0 release.
